rules-tests/CodeQuality/Rector/Assign/CombinedAssignRector/Fixture/fixture.php.inc

```diff
@@ -15,25 +15,3 @@ function combinedAssignFunction()
     $baz = $baz ** 2;
     $quux = $quux | Something::FOO;
 }
-
```

Comment:
here I am just deleting files to cleanup the output when invoking the actual test.
this does not make sense of course .. its just for repro sake.